app/uk/gov/hmrc/individualsmatchingapi/controllers/v2/PrivilegedIndividualsController.scala

```diff
@@ -43,6 +43,10 @@ class PrivilegedIndividualsController @Inject()(
     extends CommonController(cc) with PrivilegedAuthentication {
 
   def matchedIndividual(matchId: String): Action[AnyContent] = Action.async { implicit request =>
+    val redactedBody: String = request.body.toString() match {
+      case "" => request.body.toString()
+      case _ => "[Redacted Request Body]"
+    }
```

Comment:
Can you deduplicate this?

Reply:
by that do you mean extract it out into a utility class and then import it to the files that use it?

Reply:
Yes.